src/index.html

```diff
@@ -19,10 +19,36 @@
         <!--[if lt IE 8]>
             <p class="browserupgrade">You are using an <strong>outdated</strong> browser. Please <a href="http://browsehappy.com/">upgrade your browser</a> to improve your experience.</p>
         <![endif]-->
-
-        <main class="container">
-        <!-- Add your site or application content here -->
-        </main><!-- .container -->
```

Comment:
Better to _link_ to `product.html` / `listing.html` (see above) in here than put all the code for a listing in `index.html`